validator/validator.go

```diff
@@ -30,11 +29,40 @@ func Validator() *validator.Validate {
 			panic("not a felt")
 		}, felt.Felt{}, &felt.Felt{})
 		v.RegisterCustomTypeFunc(func(field reflect.Value) any {
-			if t, ok := field.Interface().(rpc.TransactionType); ok {
+			if t, ok := field.Interface().(TransactionType); ok {
 				return t.String()
 			}
 			panic("not a TransactionType")
-		}, rpc.TransactionType(0))
+		}, TransactionType(0))
 	})
 	return v
 }
+
+// TransactionType is duplicated here to prevent import cycles
```

Comment:
We really need to move the rpc types to a separate package. I'll try to send a PR for that today.

Reply:
Actually, I reverted this change locally and it appears to work now. Maybe this change was necessary at some point but is now unnecessary?

Reply:
Yup, nice catch! This is no longer needed, and has been reverted.